### PR TITLE
deps: update to `bpmn-js-properties-panel@1.22.0`

### PR DESCRIPTION
**Pull request by barmac**

### CODE REVIEW

**Comment by marstamm** [comment]
Should we update `@bpmn-io/properties-panel` to 1.8.1 in this PR or will this be handled in a separately?

**Comment by barmac** [comment]
I haven't considered that and will update that library here too. Thanks for pointing this out.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +8,13 @@ ___Note:__ Yet to be released changes appear here._
 
 ### Key changes in Properties panel
 
+* `FEAT`: support `bpmn:Message` templating in events ([#890](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/890))
+* `FEAT`: support `generatedValue` in templates ([#890](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/890))
+* `FEAT`: suggest variables for templates properties ([#904](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/904))
```
Comment (marstamm) [inline CHANGELOG.md:13]: As the issues reference another repo, let's highlight this in the linked text:

```suggestion
* `FEAT`: support `bpmn:Message` templating in events ([bpmn-js-properties-panel#890](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/890))
* `FEAT`: support `generatedValue` in templates ([bpmn-js-properties-panel#890](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/890))
* `FEAT`: suggest variables for templates properties ([bpmn-js-properties-panel#904](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/904))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,9 +8,13 @@ ___Note:__ Yet to be released changes appear here._
...
 * `FEAT`: support multiple output mappings for the same variable ([variable-resolver#15](https://github.com/bpmn-io/variable-resolver/pull/15))
 * `FEAT`: suggest variables from connector mappings ([#265](https://github.com/camunda/camunda-bpmn-js/issues/265))
 * `FEAT`: format example data preview ([example-data-properties-provider#10](https://github.com/camunda/example-data-properties-provider/pull/10))
+* `FIX`: separate Camunda 7 and BPMN concerns in Timer Props ([#910](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/910))
```
Comment (marstamm) [inline CHANGELOG.md:17]:
```suggestion
* `FIX`: separate Camunda 7 and BPMN concerns in Timer Props ([bpmn-js-properties-panel#910](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/910))
```

**Review by marstamm — APPROVED** [review]
🚀

**Pull request merged**